src/pages/SessionPage/SessionPage.jsx

```diff
@@ -100,7 +101,7 @@ export const SessionPage = () => {
               >
                 <ParticipantCard
                   label={participant.name}
-                  isSelected={currentSelection.includes(participant._id)}
+                  isSelected={currentSelection.includes(participant._id) || participant.name === window.sessionStorage.getItem("current_username")}
```

Comment:
Not needed. `currentSelection` will always have `sessionData.participantId`.

Reply:
Tested and checked this edited version to be functional and correct.